software/wmla120_ansible/anaconda_install.yml

```diff
@@ -55,5 +58,22 @@
     path: "{{ install_dir }}/.condarc"
     insertafter: '^channels:.*'
     line: "{{ item }}"
+    regexp: "{{ item }}"
   loop: "{{ ana_powerup_repo_channels }}"
+  when: not eng_mode
+  become: yes
+
+- name: "Populate IBMLAI public repo channel when eng mode"
```

Comment:
Is "IBMLAI" a typo or an initialism I'm not familiar with?

Reply:
typo